ladybug_comfort/_extend_ladybug.py

```diff
@@ -0,0 +1,130 @@
+# coding=utf-8
+"""Collection of methods that extends current objects in ladybug core library.
+
+This module is imported in __init__.py and applies the changes on runtime.
+"""
+from __future__ import division
+
+from ladybug.epw import EPW
+
+from .collection.utci import UTCI
+from .collection.pmv import PMV
+from .collection.solarcal import OutdoorSolarCal
+
+
+def get_universal_thermal_climate_index(self, include_wind=False, include_sun=False,
+                                        comfort_parameter=None):
```

Comment:
That would be very confusing since [UTCIParameter](https://github.com/chriswmackey/ladybug-comfort/blob/master/ladybug_comfort/parameter/utci.py#L8) is an object that allows you to set the criteria of what UTCI is considered comfortable.

Reply:
so what is this comfort_parameter then? It's a generic comfort parameter object?

Reply:
Ah wait. I think I completely misunderstood your original comment.  I thought you were saying that the name of the code injected method should be `UTCI_parameters` instead of `get_universal_thermal_climate_index`.

You are right that the `comfort_parameter` input is not not generic.  I will change the name of the input to `utci_parameter` then.

Reply:
Done!